### DISCUSSION TITLE
[JIT] tailwind plugins not getting loaded for a NextJS project

### DISCUSSION THREAD

**Discussion by archived-aft-bogus-german-visa-rejectio** [tailwindlabs/tailwindcss#5795] [Help]
Anyone has encountered similiar problems?

In my NextJS project, my tailwind plugins are loaded when JIT is off. But when I turn on JIT, only the basic tailwind stuff are applied. All the other classes disappear.

<img width="843" alt="Screenshot 2021-10-15 at 10 58 59 PM" src="https://user-images.githubusercontent.com/10269911/137508868-340eedf3-7742-4c08-a139-6797bd70c99d.png">

here is my `postcss.config.js` :

```
module.exports = {
    plugins: [
      'tailwindcss',
      [
        'postcss-preset-env',
        {
          stage: 1,
          features: {
            'focus-within-pseudo-class': false,
          },
        },
      ],
      // ...(process.env.NODE_ENV === "production" ? [purgecss] : [])
    ],
  }
```

**Comment by dance2die** [comment]
JIT mode requires `purge` option to be set: https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode

If it is set, a error-reproducible repo/code snippet would help me and other folks to try.  
You can fork my Stackblitz (has Next.js + TW setup) to get started too :) https://stackblitz.com/edit/next-v11?file=tailwind.config.js
